torchaudio/prototype/ctc_decoder/ctc_decoder.py

```diff
@@ -20,6 +23,9 @@
 __all__ = ["Hypothesis", "LexiconDecoder", "lexicon_decoder"]
 
 
+PretrainedFiles = namedtuple("PretrainedFiles", ["lexicon", "tokens", "lm"])
```

Comment:
Can we prefix this with underscore so that we can show it is discouraged to use this type.
This return type is introduced merely to facilitate the access, and it is not intended for the public use.